.github/workflows/build-msvc.yml

```diff
@@ -71,19 +71,49 @@ jobs:
             C:\cygwin-packages
           key: cygwin-packages
 
+      - name: Get Autoconf cache key
+        shell: bash
+        env:
+          HOST: ${{ matrix.x86_64 && 'x86_64-pc-windows' || 'i686-pc-windows' }}
+          CC: ${{ matrix.cc }}
+        run: >-
+          echo "AUTOCONF_CACHE_KEY=$HOST-$CC-$({ cat configure; uname; } | sha1sum | cut -c 1-7)" >> $GITHUB_ENV
+
+      - name: Restore Autoconf cache
+        uses: actions/cache/restore@v4
+        env:
+          AUTOCONF_CACHE_KEY: ${{ env.AUTOCONF_CACHE_KEY }}
```

Comment:
This reads like a no-op. Is that really necessary?
(This appears in 3 different steps, I point it only once)